Podcast/TrendingTopicsView.swift

```diff
@@ -0,0 +1,137 @@
+//
+//  TrendingTopicsView.swift
+//  Podcast
+//
+//  Created by Mindy Lou on 11/21/17.
+//  Copyright © 2017 Cornell App Development. All rights reserved.
+//
+
+import UIKit
+
+protocol TrendingTopicsViewDelegate: class {
+    func trendingTopicsView(trendingTopicsView: TrendingTopicsView, didSelectItemAt indexPath: IndexPath)
+}
+
+enum TrendingTopicsViewType {
+    case trending
+    case related
+}
+
+class TrendingTopicsView: UIView, UICollectionViewDelegate, UICollectionViewDataSource, UICollectionViewDelegateFlowLayout {
+
+    var iconView: ImageView!
+    var titleLabel: UILabel!
+    var descriptionLabel: UILabel!
+    var collectionView: UICollectionView!
+
+    weak var dataSource: TopicsCollectionViewDataSource?
+    weak var delegate: TrendingTopicsViewDelegate?
+
+    let titleLabelText = "Trending Topics"
+    let descriptionLabelText = "Find podcasts that everyone is talking about."
+    let iconViewBorderPadding: CGFloat = 20
+    let iconViewLength: CGFloat = 24
+    let iconViewContentPadding: CGFloat = 10
+    let titleDescriptionLabelPadding: CGFloat = 9
+    let descriptionLabelTopOffset: CGFloat = 7.5
+    let descriptionCollectionViewPadding: CGFloat = 18
+    let collectionViewTopOffset: CGFloat = 14.5
+    let collectionViewBottomInset: CGFloat = 24
+    let titleLabelHeight: CGFloat = 18
+    let descriptionLabelHeight: CGFloat = 17
+
+    let reuseIdentifier = "Cell"
+
+    init(frame: CGRect, type: TrendingTopicsViewType) {
+        super.init(frame: frame)
+        backgroundColor = .offWhite
+
+        iconView = ImageView(frame: CGRect(x: 0, y: 0, width: iconViewLength, height: iconViewLength))
+        iconView.image = #imageLiteral(resourceName: "trending")
+        addSubview(iconView)
+        iconView.snp.makeConstraints { make in
+            make.top.leading.equalToSuperview().offset(iconViewBorderPadding)
+            make.width.height.equalTo(iconViewLength)
+        }
+
+        titleLabel = UILabel()
+        titleLabel.font = ._20SemiboldFont()
+        addSubview(titleLabel)
+        titleLabel.snp.makeConstraints { make in
```

Comment:
![image](https://user-images.githubusercontent.com/34046832/37006242-77e01d9e-20a5-11e8-84ac-47bec4d227c1.png)
Decrease padding so it's aligned with the Top Series below